docs/scripts/Pipfile

```diff
@@ -8,6 +8,7 @@ verify_ssl = true
 [packages]
 sphinx-autobuild = "*"
 sphinx = "*"
+jinja2 = ">=2.10"
```

Comment:
The vulnerability report said we specifically want `>2.10`, or `>=2.10.1`, I believe.

Reply:
good spot @garyverhaegen-da -- repushed with uptick to >=2.10.1